ted_adapter/test_adapter.py

```diff
@@ -119,6 +158,13 @@ def main():
     :return: None
     """
 
+    if not driver_exists():
+        path = str(adapter_conf.DRIVER_PATH)
+        print(f"Driver was not found in this location: {path}")
```

Comment:
"...: %s" % str(adapter_conf.DRIVER_PATH) # probably str is not needed too
also should be in the function